### ISSUE TITLE
Confusion with user account numbering

### ISSUE THREAD

**Issue by ghost** [GabyPCgeeK/savemii#2]
There seem to be a confusion about user accounts and what folder the system creates. I have three user accounts on my system and they are as follows
80000001, 80000002, 80000006

Like how'd it jump that far ahead for user three! And I met someone who just uses one user account and his was 80000008!

So I was helping him to restore a save he got somewhere. And I had him name it 80000001 because I assumed it would be that. But the save failed to restore. So we backed up his save just compare the files. Once we renamed the folder to 80000008, it restored successfully.

This kind've a problem, I wish there was a better way to know the user number. Maybe you can have savemii import the save regardless of the folder name.

**Comment by GabyPCgeeK** [comment]
Yes I'm currently working on "cross account restore" that will have you select account folder from the selected slot and select what account to restore it to. Right now you can view the Persistent ID (ex 80000001) for the account running the homebrew by selecting a game in savemii, going into the backup/restore menu and on "Select user:" select "this user" and to the right the Persistent ID will appear. (ex < this user > 80000008)

From what I've seen in my own WiiU, some research I've done and from the fact that the number is called Persistent ID it seems that every time a new account is created it is assigned a Persistent ID equal to last created account's Persistent ID +1 (Doesn't matter if you deleted accounts prior creating the account). So probably you created 3 account and deleted them before/after creating the last one and in the 80000008 case there were 7 accounts before the 80000008 one.

If that is not what happened in your case please let me know.

**Comment by ghost** [comment]
@GabyPCgeeK Haha nope, I've only created two additional user accounts without deleting any accounts. I've never deleted any user account before, in fact I just noticed the delete user button yesterday for the first time. I'm like "ah that's where you are, and how you can be dangerous for CBHC!"

I love your work on Savemii, keep it up man!
